README.md

```diff
@@ -33,21 +33,37 @@ git clone git@github.com:jonathanrocher/ets_tutorial.git
 
 ### EDM users (recommended)
 First, download and install EDM from https://www.enthought.com/edm/. Then, 
-open a `Terminal`/`Powershell`/`Cmd Prompt`/ and navigate to the folder 
-where the repo was cloned. Enter the following command to create a 
-dedicated Python environment and install all dependencies in it:
+open a `Terminal`/`Powershell`/`Cmd Prompt`/ and create a lighweight bootstrap environment to run the installation commands.
 ```commandline
-python ci build
+edm envs create bootstrap
+edm install --environment bootstrap click
+```
+Next, enter the following commands to create a 
+dedicated Python environment called `ets_tutorial` and install all dependencies in it:
+```commandline
+edm run -e bootstrap -- python ci build --environment ets_tutorial
+```
+All application related `python` commands are assumed to run in this
+environment. You can activate the environment with:
+```commandline
+edm shell -e ets_tutorial
 ```
 
 ### Conda users
-[TODO]
+Create a new conda environment called `ets_tutorial` and install local
+dependencies with the following commands:
+```commandline
+conda create -n ets_tutorial python=3.6 pandas matplotlib traits traitsui scikit-image pillow pyqt ipython importlib_resources importlib_metadata
+python setup.py install
```

Comment:
You should have them activate the environment first -- otherwise ets_tutorial will be installed in their base environment, not in the `ets_tutorial` environment.

Reply:
Great catch! Also, hello! Nice to meet you, and apologies for not introducing ourselves earlier. We're a little bit behind, as you could probably tell from the recent flurry of PRs!